pkg/detect/types/detect.go

```diff
@@ -30,6 +30,21 @@ type VulnerabilityData struct {
 }
 
 type VulnerabilityDataDetection struct {
-	Ecosystem ecosystemTypes.Ecosystem                                           `json:"ecosystem,omitempty"`
-	Contents  map[string]map[sourceTypes.SourceID]criteriaTypes.FilteredCriteria `json:"contents,omitempty"`
+	Ecosystem ecosystemTypes.Ecosystem                                             `json:"ecosystem,omitempty"`
+	Contents  map[string]map[sourceTypes.SourceID]conditionTypes.FilteredCondition `json:"contents,omitempty"`
+}
+
+func (r *DetectResult) Sort() {
```

Comment:
In what situations is this sort intended to be used?

Reply:
Not needed yet. will remove

Reply:
https://github.com/MaineK00n/vuls2/compare/296a7fd45df1a2a7c22523a475d4b19cb089b837..edd42367d12bf3ac9083eb390e4cfb4e0271ace6